Neos.Eel/Tests/Unit/Helper/DateHelperTest.php

```diff
@@ -52,7 +53,7 @@ public function formatExamples()
         return [
             'DateTime object' => [$dateTime, 'Y-m-d H:i:s', '2013-07-03 12:34:56'],
             'timestamp as integer' => [1372856513, 'Y-m-d', '2013-07-03'],
-            'timestamp as string' => ['1372856513', 'Y-m-d', '2013-07-03'],
+            'timestamp as string' => ['@1372856513', 'Y-m-d', '2013-07-03'],
```

Comment:
Scrap that. Indeed a string like `20160101` is parsed as a date and would still be numeric, leading to very strange behaviour. Talk about magic…

So my vote: drop string handling. If people want to parse a string, why not let them use `DateTime` themselves? Using a proper object to pass dates around feels better anyway.